lib/watir-webdriver/wait.rb

```diff
@@ -114,18 +107,44 @@ def respond_to?(*args)
       @element.respond_to?(*args)
     end
 
-    def method_missing(m, *args, &block)
+    def method_missing(decorator, m, *args, &block)
```

Comment:
I totally agree with you here on this. I made the change. If you or Titus wants me to change it back, it's easy enough to do. I've made all of your suggestions and because rebasing was proving to be convoluted I've opened fresh PR's for both watir-webdriver (https://github.com/watir/watir-webdriver/pull/370) and watirspec (https://github.com/watir/watirspec/pull/72).